contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -341,9 +343,8 @@ contract LoanTokenLogicStandard is LoanTokenSettingsLowerAdmin {
 
 	/* Public View functions */
 
-	function profitOf(address user) public view returns (uint256) {
-		// keccak256("iToken_ProfitSoFar")
-		bytes32 slot = keccak256(abi.encodePacked(user, uint256(0x37aa2b7d583612f016e4a4de4292cb015139b3d7762663d06a53964912ea2fb6)));
+	function profitOf(address user) public view returns (int256) {
```

Comment:
victor is using the function on the frontend and this changes the abi slightly. if you're changing any public functions, Victor needs to know about it.